08-configs.md

```diff
@@ -1,46 +1,50 @@
 ## Configs top-level element
 
-Configs allow services to adapt their behaviour without the need to rebuild a Docker image. Configs are comparable to Volumes from a service point of view as they are mounted into service's container's filesystem. The actual implementation detail to get the configuration provided by the platform can be set from the Configuration definition.
```

Comment:
Check: I removed the second sentence because I'm pretty sure it's not necessary and is explained more clearly further down the page.